test/system/src/test/java/io/pravega/test/system/BookieFailoverTest.java

```diff
@@ -188,42 +188,48 @@ public void bookieFailoverTest() throws ExecutionException, InterruptedException
             createWriters(clientFactory, NUM_WRITERS, SCOPE, STREAM);
             createReaders(clientFactory, readerGroupName, SCOPE, readerGroupManager, STREAM, NUM_READERS);
 
-            //scale down bookie
+            // Give some time to create readers before forcing a bookie failover.
+            Exceptions.handleInterrupted(() -> Thread.sleep(WAIT_AFTER_FAILOVER_MILLIS));
+
+            // Scale down bookie.
             Futures.getAndHandleExceptions(bookkeeperService.scaleService(2), ExecutionException::new);
 
-            log.info("Sleeping for 1 min");
-            Exceptions.handleInterrupted(() -> Thread.sleep(1 * 60 * 1000));
+            log.info("Sleeping for {} seconds.", WAIT_AFTER_FAILOVER_MILLIS / 1000);
+            Exceptions.handleInterrupted(() -> Thread.sleep(WAIT_AFTER_FAILOVER_MILLIS));
 
-            long writeCounteBeforeSleep  = testState.getEventWrittenCount();
-            log.info("Write count after bookie failover after 1 min sleep {}", writeCounteBeforeSleep);
+            long writeCountBeforeSleep  = testState.getEventWrittenCount();
+            log.info("Write count after bookie failover after {} seconds sleep {}.", WAIT_AFTER_FAILOVER_MILLIS / 1000, writeCountBeforeSleep);
```

Comment:
Can you make it `Write count after {} seconds sleep after bookie failover is: {}` or 
`Write count is {} after {} seconds sleep after bookie failover. ` 
Just to avoid confusion of the two parameters one after and before the sleep

Reply:
Log message changed..